outline.md

```diff
@@ -0,0 +1,39 @@
+# Outline File
+
+This is used to outline plans for the guide.
+
+Each page of the guide is a separate file marked in the outline with a number.
+
+If two pages are very similar they can share a number (1a, 1b). For example, some pages from the CSS guide and JS Guide will be similar to each other with slight differences.
+
+Numbers under 10 are high priority.
+
+Items marked with a start * require further discussion.
+
+## CSS
+ - Welcome (1)
+ - Installing Mods (2)
+ - Inspecting the interface (3a)
+ - Your first CSS Mod (6)
+ - Using Browser Classes (8)
+ - Sharing is Caring! : Posting Mods and Best Practices * (4)
+
+
+
+## JS
+- Welcome (1)
+- Installing Mods (2)
+- Inspecting the interface (3b)
+ - And using the Console
+- Your First JS Mod (5)
+- Keyboard Shortcuts (9)
```

Comment:
What for? *Modding* keyboard shortcuts, or some (?) ‘standard set’ that might be useful?

Reply:
Lonm’s keyboard machine is very useful for modding because you can call your own functions. It wouldn’t take long to explain.

Reply:
I’d move it into *advanced JS* then.

Reply:
👌